stages/eib_image

```diff
@@ -155,11 +155,12 @@ if [[ "${EIB_IMAGE_SDBOOT}" = "true" ]]; then
   unset sdboot_esp_img_loop
 fi
 
+# FIXME: need a manifest in a new way
 # Output package list using the dpkg database from the deployment's
 # immutable copy in /usr/share/dpkg/database.
-dpkgdir="${OSTREE_DEPLOYMENT}"/usr/share/dpkg/database
-packageinfo="${EIB_TMPDIR}"/packages.txt
-dpkg-query -W --admindir="${dpkgdir}" > "${packageinfo}"
+#dpkgdir="${OSTREE_DEPLOYMENT}"/usr/share/dpkg/database
+#packageinfo="${EIB_TMPDIR}"/packages.txt
+#dpkg-query -W --admindir="${dpkgdir}" > "${packageinfo}"
```

Comment:
Please delete this instead of leaving commented lines that won't work anymore.

Reply:
Cleaned this up in 34a89ddaf3c80f8a78c302e516241410cfa0aa0c